src/interfaces.ts

```diff
@@ -20,6 +20,7 @@ export enum ProtocolType {
   UNKNOWN = '<unknown>',
   STDIO = 'stdio',
   TCP = 'tcp',
+  DOCKER = 'docker'
```

Comment:
This may not be the _right_ way to do things but it enabled us to do some easy plumbing. Could probably be reworked?